### ISSUE TITLE
AttributeError: 'Box' object has no attribute 'n'

### ISSUE THREAD

**Issue by Jaccccc** [pnp/generator-spfx#236]
I have just started coding ai with gym, im using the gym import and gym_tetris, and i keep getting th error that "AttributeError: 'Box' object has no attribute 'n'". Any explanation would be great, but bear in mind i am pretty new to all this, so you may have to dumb it down a bit!
The line that was making the error occur was : env.observation_space.n

**Comment by StfBauer** [comment]
For all gym related question please open an issue on the [gym issue list](https://github.com/openai/gym/issues).
